packages/mobile/package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "android": "react-native run-android",
     "ios": "react-native run-ios",
+    "ipad": "react-native run-ios --simulator='iPad Air (4th generation)'",
```

Comment:
Can we avoid adding this script? You can run it anyway passing the simulator options to `yarn ios` 👍